src/wallet/CMakeLists.txt

```diff
@@ -121,7 +121,6 @@ if (BUILD_GUI_DEPS)
             wallet
             rpc_base
             multisig
-            blockchain_db
```

Comment:
`cryptonote_core` depends on `blockchain_db`, so we have to link `blockchain_db` as well

Reply:
Does the GUI wallet depend on cryptonote_core ? I don't see explicit deps.

Reply:
GUI depends on `wallet_api`, `wallet_api` depends on `cryptonote_core`

Reply:
OK, then I'll leave it to someone who builds the GUI.